### DISCUSSION TITLE
you can open nonexistent discussions pages in the app

### DISCUSSION THREAD

**Discussion by 3m4r5** [community/community#18253] [Mobile] [answered]
example: [the atom repo discussions](https://github.com/atom/atom/discussions)
[screen recording](https://user-images.githubusercontent.com/63432932/173047626-6d16f842-5a30-4d13-b8e3-5485c06c1029.MOV)

**Comment by brianlovin** [comment, marked as answer]
Got it, thank you for the heads up! Yeah we should probably fall back to a more clear empty state on a 404 view. I'll open an issue and pass it along to the team :)

**Reply by 3m4r5** [reply]
Great 👍
